pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -766,7 +766,7 @@
     "sensor.cups" = ps: with ps; [ pycups ];
     "sensor.currencylayer" = ps: with ps; [  ];
     "sensor.daikin" = ps: with ps; [  ];
-    "sensor.darksky" = ps: with ps; [  ];
+    "sensor.darksky" = ps: with ps; [ python-forecastio ];
```

Comment:
This needs to be a separate commit.

Reply:
as discussed above, this should be done somehow automatically by the update script but it doesn't for some reason

Reply:
I have just tried running the ```parse-requirements.py``` script here and it works perfectly fine. In addition to updating the darksky ```sensor``` component it also updates the ```weather``` component as expected. What happens when you run it?

As mentioned by @FRidh that needs to go into a separate commit.

Reply:
`parse-requirements.py` works for me as well. You should however rebase onto latest master, where Home Assistant was updated to 0.72.0.